config/samples/agent-install.openshift.io_v1beta1_agentserviceconfig.yaml

```diff
@@ -1,58 +1,37 @@
 apiVersion: agent-install.openshift.io/v1beta1
 kind: AgentServiceConfig
-metadata:
-  name: agent
+metadata: {name: agent}
```

Comment:
This is unnecessary and `verify-generated-code` doesn't like it

Reply:
Reverted this change and the other seemingly unnecessary YAML reformatting changes that the 'make generate-configuration' command did, but kept the new P/Z osImages parameters. [Commit](https://github.com/openshift/assisted-service/pull/4333/commits/794af8173723ee8c6e12072b53cf7aa04445022d).